myia/frontends/pytorch_functions.py

```diff
@@ -19,6 +19,7 @@
 from functools import reduce
 
 import torch
+import torch.nn.functional
```

Comment:
you don't need to import torch.nn.functional

Reply:
Done!